docs/attachments/docker-plugin-configuration-script.groovy

```diff
@@ -56,11 +58,13 @@ DockerTemplateBase dockerTemplateBase = new DockerTemplateBase(
   dockerTemplateBaseParameters.volumesFromString,
   dockerTemplateBaseParameters.environmentsString,
   dockerTemplateBaseParameters.hostname,
+  dockerTemplateBaseParameters.user,
```

Comment:
The constructor used here was deprecated in 2017; we need to stop using it.
Call `new DockerTemplateBase(dockerTemplateBaseParameters.image)` and then set all the remaining parameters.
...or try something like `dockerTemplateBaseParameters as DockerTemplateBase` or similar groovy "magic" to set everything.

...and either remove or update the `// https://github.com/jenkinsci/docker-plugin/blob/docker-plugin-1.1.2/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java` comment that's pointing to an ancient version of the plugin.

Reply:
I agree if the DockerTemplateBase constructor is deprecated it makes sense to update the script properly. My groovy skills do not go far enough for that.
I hit upon the script from the official docs but it just errored out.

Reply:
hey @adamjwsuch, I have created a [PR](https://github.com/adamjwsuch/docker-plugin/pull/1) in your fork which addresses deprecation stuff.

Would you be able to review and merge it, please?